testers/testers/java/bin/install.sh

```diff
@@ -16,8 +16,7 @@ compile_tester() {
 
 update_specs() {
     echo "[JAVA-INSTALL] Updating specs"
-    cp ${TESTERDIR}/specs/default_install_settings.json ${TESTERDIR}/specs/install_settings.json
-    sed -i -e "s#/path/to/tester/jars#${JAVADIR}/build/install/MarkusJavaTester/lib#g" ${TESTERDIR}/specs/install_settings.json
+    echo '{}' | jq ".path_to_tester_jars = \"${JAVADIR}/build/install/MarkusJavaTester/lib\"" > ${TESTERDIR}/specs/install_settings.json
```

Comment:
In general I prefer to use `jq` instead of `sed` because it will handle the case where things we're trying to replace are split over several lines.  It's not really necessary in this case since we're creating a json string from scratch but I mostly did it to be consistent with other `jq` operations.  

So not necessary but I like it as good practice in general.